src/components/admin/event/ManageEventCard/index.tsx

```diff
@@ -85,6 +137,7 @@ const ManageEventCard = ({ event }: IProps) => {
       <Link href={`${config.admin.events.editRoute}/${event.uuid}`}>Edit Details</Link>
       <Button onClick={duplicateEvent}>Duplicate Event</Button>
       <Button onClick={generateDiscordEvent}>Generate Discord Event</Button>
```

Comment:
Can we have this be Generate Discord Event / Sync Discord Event depending on if DiscordEvent is set?